popSimManu.tex

```diff
@@ -221,7 +221,7 @@ \section*{Introduction}
 This resource, which we call \stdpopsim, makes running
 realistic simulations for population genetic analysis a simple matter of
 choosing pre-implemented models from a community-maintained catalog.
-The \stdpopsim catalog currently contains five organisms: humans, \emph{Pongo~pygmaeus},
+The \stdpopsim catalog currently contains four organisms: humans,
```

Comment:
Currently Figure 1 and Table 1 only have 3 taxa.

Reply:
okay i can make it just 3 for the paper